imap_processing/hi/l1a/hi_l1a.py

```diff
@@ -32,14 +33,17 @@ def hi_l1a(packet_file_path: str):
             # TODO: Add processing for science count data
             continue
         elif apid == HIAPID.H45_SCI_DE:
-            logging.debug(
+            logging.info(
```

Comment:
We should get a logger in this module to follow the other instruments.
```suggestion
            logger.info(
```